src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/PackagingTest.cs

```diff
@@ -311,6 +311,32 @@ public void CheckSignApk ([Values(true, false)] bool useApkSigner, [Values(true,
 			}
 		}
 
+		[Test]
+		public void CheckAppBundle ()
+		{
+			var proj = new XamarinAndroidApplicationProject () {
+				IsRelease = true,
```

Comment:
Is `.aab` output a "sensible" thing for Debug builds?  If yes, shouldn't we check both?  If not, did we assert that anywhere in 4f088670f2418d6291bd6d4b41fea0dd722081fb?  (I don't see any new error checks...)